Content.Server/Flight/FlightSystem.cs

```diff
@@ -0,0 +1,151 @@
+
+using Content.Shared.Cuffs.Components;
+using Content.Shared.Damage.Components;
+using Content.Shared.DoAfter;
+using Content.Shared.Flight;
+using Content.Shared.Flight.Events;
+using Content.Shared.Mobs;
+using Content.Shared.Popups;
+using Content.Shared.Stunnable;
+using Content.Shared.Zombies;
+using Robust.Shared.Audio.Systems;
+
+namespace Content.Server.Flight
+{
```

Comment:
```suggestion
namespace Content.Server.Flight;
```
Just like before, you're going to need to drop the entire file down a tab indent.